src/components/Pages/Home.tsx

```diff
@@ -1,19 +1,82 @@
 import React from 'react';
 import Categories from '../Categories/Categories';
+import TemporaryDrawer from '../Categories/CategoriesMenu';
 import CdList from '../CdList/CdList';
 import Footer from '../Footer/Footer';
 import Discount from '../Header/Discount';
 import Header from '../Header/Header';
+import ButtonAppBar from '../Header/Navbar';
 import MainContent from '../MainContent/MainContent';
 import Newsletter from '../Newsletter/Newsletter';
+import Box from '@mui/material/Box';
+import Drawer from '@mui/material/Drawer';
+import Button from '@mui/material/Button';
+import List from '@mui/material/List';
+import Divider from '@mui/material/Divider';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import InboxIcon from '@mui/icons-material/MoveToInbox';
+import MailIcon from '@mui/icons-material/Mail';
 
 const Home = () => {
+
+	type Anchor = 'left';
+
+
+
+
+ function TemporaryDrawer() {
+	const [state, setState] = React.useState({
+		left: false,
+	});
+
+
+	const toggleDrawer =
+		(anchor: Anchor, open: boolean) =>
+		(event: React.KeyboardEvent | React.MouseEvent) => {
+			if (
+				event.type === 'keydown' &&
+				((event as React.KeyboardEvent).key === 'Tab' ||
+					(event as React.KeyboardEvent).key === 'Shift')
+			) {
+				return;
+			}
+
+			setState({ ...state, [anchor]: open });
+		};
+
+	const list = (anchor: Anchor) => (
+		<Box
+			sx={{ width: 250 }}
+			role='presentation'
+			onClick={toggleDrawer(anchor, false)}
+			onKeyDown={toggleDrawer(anchor, false)}>
+			<List>
+				{['HipHop', 'Metal', 'Pop', 'Rock'].map((text, index) => (
+					<ListItem key={text} disablePadding>
+						<ListItemButton>
+							<ListItemIcon>
+								{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}
+							</ListItemIcon>
+							<ListItemText primary={text} />
+						</ListItemButton>
+					</ListItem>
+				))}
+			</List>
+		</Box>
+	);
+
+
```

Comment:
the code is identical as TemporaryDrawer in CategoriesMenu - do not duplicate code

Reply:
reuse it instead :)